packages/core/src/ui/UICanvas.ts

```
@@ -0,0 +1,619 @@
+        case CanvasRenderMode.ScreenSpaceOverlay:
+          this._removeCanvasListener();
+        case CanvasRenderMode.ScreenSpaceCamera:
+        case CanvasRenderMode.WorldSpace:
+          componentsManager.removeUICanvas(preRealMode, this);
+          break;
+        default:
+          break;
+      }
+      switch (curRealMode) {
+        case CanvasRenderMode.ScreenSpaceOverlay:
+          this._addCanvasListener();
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)